pkg/scheduler/worker/workerpool_test.go

```diff
@@ -2,6 +2,10 @@ package worker
 
 import (
 	"context"
+	test2 "github.com/kyma-incubator/reconciler/pkg/keb/test"
```

Comment:
I find this name pretty confusing, as if there was a `test` package also imported. I would either call it something descriptive like `kebTest`.

Reply:
That is true! Sorry, I wasnt looking at the auto-generated code in Goland. I will take more care about such stuff in future!